code/modules/research/designs/mechfabricator_designs.dm

```diff
@@ -793,18 +793,18 @@
 	category = list("Exosuit Equipment")
 
 /datum/design/mech_ccw_armor
-	name = "Exosuit Module (Reactive Armor Booster Module)"
+	name = "Exosuit Module (Melee Armor Booster Module)"
```

Comment:
Please note these name changes in the PR description

Reply:
got it